src/openzaak/conf/includes/base.py

```diff
@@ -442,6 +443,11 @@
 PROJECT_NAME = "Open Zaak"
 SITE_TITLE = "API dashboard"
 
+# if specified, this replaces the host information in the requests, and it is used
+# to build absolute URLs.
+OPENZAAK_DOMAIN = config("OPENZAAK_DOMAIN", "")
+OPENZAAK_REWRITE_HOST = config("OPENZAAK_REWRITE_HOST", False)
```

Comment:
I've split this into two settings, one affects the middleware, the other is always used if it's available to build requests in non-HTTP contexts (such as exports, notifications...) to eventually replace `Site.objects.get_current()` usage.